docs/contribute/cumulative-docs/guidelines.md

```diff
@@ -0,0 +1,263 @@
+and the [feature lifecycle](/contribute/cumulative-docs/reference.md#lifecycle)
+and [version](/contribute/cumulative-docs/reference.md#version) are make up the value.
+
+```
+<key>: <lifecycle> <version>
+```
+
+### Level
+
+For each type of applicability information, you can add `applies_to` metadata at different levels:
+
+* **Page-level** metadata is **mandatory** and must be included in the frontmatter.
+  This defines the overall applicability of the page across products, deployments, and environments.
+* **Section-level** annotations allow you to specify different applicability for individual sections
+  when only part of a page varies between products or versions.
+% TO DO: Add when https://github.com/elastic/docs-builder/issues/1436 is complete
+% * **Element-level** annotations allow tagging block-level elements like tabs, dropdowns, and admonitions.
+%  This is useful for ...
+* **Inline** annotations allow fine-grained annotations within paragraphs or definition lists.
+  This is useful for highlighting the applicability of specific phrases, sentences,
+  or properties without disrupting the surrounding content.
+
+For a full syntax reference for page, section, and inline level `applies_to` annotations,
+refer to [](/syntax/applies.md).
+
+% TO DO: Can these be pruned? 🌿
+## General guidelines
+
+% Source: Used to be in `docs/syntax/applies.md`
+* Use `applies_to` tags when features change state (`preview`, `beta`, `ga`, `deprecated`, `removed`) or when
+  availability differs across deployments and environments.
+* Use `applies_to` tags to indicate which product or deployment type the content applies to. This is mandatory for every
+  page.
+* Use `applies_to` tags when features change state in a specific update or release.
+* Do _not_ tag content-only changes like typos, formatting, or documentation updates that don't reflect feature lifecycle
+  changes.
+* You do _not_ need to tag every section or paragraph. Only do so if the context or applicability changes from what has
+  been established earlier.
+* If the product is not versioned (meaning all users are always on the latest version, like in serverless or cloud), you
+  do _not_ need to tag a new GA feature.
+
+% Source: Slack conversation
+* The `applies_to` badges should not require contributors to add specific Markdown real estate
+  to the page layout (such as a specific `Version` column in a table).
+  Instead, contributors should be able to add them anywhere they need, and the system should
+  be in charge of rendering them clearly.
+
+% Source: https://github.com/elastic/kibana/pull/229485/files#r2231850710
+% * Create hierarchy of versioned information??
+
+% Source: George's checklist
+* Don’t overload with exclusions unless it is necessary - If a page is only about Elastic Cloud Hosted, no need to say serverless: unavailable, just add deployment: ess:
+
+% TO DO: Open an issue to force the order in the code.
```

Comment:
@bmorelli25 https://github.com/elastic/docs-builder/pull/1727 right?

Reply:
Yes. Brandon beat me to it after-hours 🙂